src/pages/ProductsPage/Products.tsx

```diff
@@ -1,37 +1,40 @@
 import "./products.scss";
 import React, { useState, useEffect } from "react";
 import Product from "../../components/ui/productCard/ProductCard";
+import Loader from "../../components/ui/loader/Loader";
 import { BreedType } from "../../types";
 
 export default function Products() {
-  const [jsonInfo, setInfo] = useState<BreedType[]>([]);
-  const [errorMessage, setError] = useState<string>("");
-  const max = 21;
+  const [breed, setBreed] = useState<BreedType[]>([]);
+  const numberOfCards = 21;
   useEffect(() => {
     fetch("https://api.thedogapi.com/v1/breeds?limit=25&page=0")
       .then((res) => res.json())
-      .then((data) => setInfo(data))
+      .then((data) => setBreed(data))
       .catch((error) => {
         console.error("Error fetching breed information:", error);
-        setError(String(error.errorMessage));
       });
-  }, [errorMessage]);
+  }, [breed]);
   return (
     <div className="products container-fluid">
       <h1 className="products__title">product page</h1>
-      {errorMessage &&
-        "Oops, something went wrong. Error message: " + errorMessage}
-      <div className="row products-page">
-        {jsonInfo.map((dogy, index) => (
-          <div
-            key={dogy.id}
-            className="col-12 col-md-6 col-lg-3 products-page_card"
-            style={index >= max - 1 ? { display: "none" } : undefined}
-          >
-            <Product dogy={dogy} limit={max} />
-          </div>
-        ))}
-      </div>
+      {breed.length ? (
+        <div className="row products-page">
+          {breed.map((dogy, index) => (
+            <div
+              key={dogy.id}
+              className="col-12 col-md-6 col-lg-3 products-page_card"
```

Comment:
products-page_card should be renamed to products-page__card since it's child of products-page root class